web_text_widget/static/src/js/text_limited.js

```diff
@@ -0,0 +1,56 @@
+openerp.web_text_widget = function (instance)
+{
+
+var QWeb = instance.web.qweb;
+var _t = instance.web._t;
+
+
+instance.web.form.FieldText.include({
+    template: 'FieldText',
+    LIMIT_LINES_CONTEXT_KEY: 'limit_lines',
+    LIMIT_LINES_DEFAULT: 10,
+    LIMIT_CHARS_CONTEXT_KEY: 'limit_chars',
+    LIMIT_CHARS_DEFAULT: 500,
```

Comment:
Extending the standard widget, these default should be 0, meaning for the code below that no limit is to be applied.